### PR TITLE
removed /bugreport from csc /help output (closes #14171)

### LINKED ISSUE #14171: Please remove /bugreport from the output of csc /help
csc does not support the /bugreport command-line option, so we should probably stop teasing users by documenting it in the /help output.

### PR DESCRIPTION
**Pull request by jaredpar**
**Customer scenario**

Removes unsupported parameter from compiler output. 

**Bugs this fixes:**

#14171

**Risk**

Very low

### CODE REVIEW

**Comment by jaredpar** [comment]
CC @dotnet/roslyn-compiler for review

File: src/Compilers/CSharp/Portable/CSharpResources.resx [hunk 1]
```diff
@@ -4488,7 +4488,6 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
 
                         - ADVANCED -
  /baseaddress:&lt;address&gt;        Base address for the library to be built
- /bugreport:&lt;file&gt;             Create a 'Bug Report' file
```
Comment (jcouv) [inline src/Compilers/CSharp/Portable/CSharpResources.resx]: Consider removing from VBResources.resx too. The option is no longer supported in VB either.

Comment (jaredpar) [inline src/Compilers/CSharp/Portable/CSharpResources.resx]: done

**Comment by jaredpar** [comment]
CC @MeiChin-Tsai for ask mode approval

**Comment by MeiChin-Tsai** [comment]
approved

**Pull request merged**